deployment/kubernetes/aws-eks/tf-cluster/Makefile

```diff
@@ -14,4 +14,4 @@ kubeconfig:
 config-map-aws-auth:
 	terraform output config-map-aws-auth > config-map-aws-auth.yaml
 	kubectl apply -f config-map-aws-auth.yaml
-	watch -n 1 kubectl get nodes
+	until kubectl get nodes | grep -m 2 "Ready"; do sleep 1 ; done	
```

Comment:
would be better to not use the `-m 2` and have a way to check all instances listed are ready